plugins/radar-android-google-sleep/src/main/java/org.radarbase.passive.google.sleep/GoogleSleepManager.kt

```diff
@@ -54,6 +55,7 @@ class GoogleSleepManager(context: GoogleSleepService) : AbstractSourceManager<Go
         name = context.getString(R.string.googleSleepDisplayName)
         sleepBroadcastReceiver = SleepReceiver(this)
         sleepPendingIntent = createSleepPendingIntent()
+        registerSleepReceiver()
```

Comment:
This should happen in `start` in `sleepHandler.execute` before registering for sleep data.